### PR TITLE
feat: Add ability to inverse mask

### PR DESCRIPTION
**Pull request by katmai7**
##### Description of change
Add ability to inverse mask.

Done:
- [x] Inverse for stencil mask
- [x] Inverse for alpha mask

##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

Related issues:
https://github.com/pixijs/pixijs/issues/8207

### CODE REVIEW

File: src/filters/mask/mask.wgsl [hunk 1]
```diff
@@ -79,20 +77,22 @@ fn mainFragment(
 
     var maskClamp = filterUniforms.uMaskClamp;
 
-     var clip = step(3.5,
-        step(maskClamp.x, filterUv.x) +
-        step(maskClamp.y, filterUv.y) +
-        step(filterUv.x, maskClamp.z) +
-        step(filterUv.y, maskClamp.w));
+    var clip = step(3.5,
```
Comment (enriko-riba) [inline src/filters/mask/mask.wgsl:80]: clip is unused?

Comment (katmai7) [reply src/filters/mask/mask.wgsl:80]: Oops, I mistakenly removed it from shader.. will return it back

Comment (katmai7) [reply src/filters/mask/mask.wgsl:81]: Fixed

**Comment by bigtimebuddy** [comment]
Absolutely love this feature. Super useful and is a think that folks often ask for.

Had a chat about this with the team, in terms of the Container API, we'd like to avoid `maskInverse`. We had this similar problem in v7 with `cacheAsBitmap` having to add a bunch of properties like `cacheAsBitmapResolution`, `cacheAsBitmapMultisample`, etc. It's kinda clunky.

Our preference would be to add a `setMask(options)` method---`options` would have `mask` and `inverse` properties. In the future, I could see us adding more pass-thru customizations like `resolution` (for mask filters). Also, we should keep the `mask` setter in Container for backward compatibility and convenience.

**Comment by bigtimebuddy** [comment]
Yeah, setMask as a new method on container.

**Review by GoodBoyDigital — APPROVED** [review]
thanks so much for this @katmai7 !!! we have wanted this feature for ages :D

**Pull request merged**